### PR TITLE
Added Active Balance to Account Header

### PR DESCRIPTION
**Pull request by Ncookiez**
If the user's active balance is different from their deposited balance, the account page will now display their active balance.

For example, this lucky guy/gal just got delegated $50k from the No Loss November event:

image

The balance will be displayed (if necessary) once the deposited balance is fully calculated, and animates upwards in the same manner.

If we want to disable this at any point, we can just remove `showActive` from any page/view/component calling `AccountCard`.

### CODE REVIEW

**Comment by dylandesrosier** [comment]
We should add a tooltip explaining what "active" means. Maybe a little `opacity-80` on the "Active: $50,150" portion to make it secondary as well. Otherwise it's looking good!

File: src/views/Account/AccountCard/index.tsx [hunk 1]
```diff
@@ -80,6 +115,17 @@ export const TotalBalanceAmount: React.FC<{ usersAddress: string }> = (props) =>
   return <CountUp countTo={Number(balancesData?.totalV4Balance)} />
 }
 
+export const ActiveBalance: React.FC<{ value: number }> = (props) => {
+  const { value } = props
+  const { t } = useTranslation()
+  return (
+    <span className='font-semibold text-xs'>
+      {t('active')}:&nbsp;$
```
Comment (dylandesrosier) [inline src/views/Account/AccountCard/index.tsx:123]: We could also just use `formatCurrencyNumberForDisplay` for simplicity. Once we add multiple currencies we'll thank ourselves for that. I find too many of the rolling numbers to be distracting.

Comment (Ncookiez) [reply src/views/Account/AccountCard/index.tsx]: Resolved with 4cc42e9

**Comment by Ncookiez** [comment]
Made the suggested updates!

The tooltip text is under the key `activeBalanceTooltip` for any future changes :)

**Pull request closed without merging**